atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -240,7 +241,7 @@ private CASResult writeLockWithCas(
     private Column lockColumnWithValue(byte[] value) {
         return new Column()
                 .setName(CassandraKeyValueServices.makeCompositeBuffer(
-                        CassandraConstants.GLOBAL_DDL_LOCK_COLUMN_NAME.getBytes(),
+                        CassandraConstants.GLOBAL_DDL_LOCK_COLUMN_NAME.getBytes(StandardCharsets.UTF_8),
```

Comment:
Using PtBytes is preferable for these conversions since that's what we typically do on other column name conversion paths, and that implicitly uses UTF-8 charset.